src/DataTable/DropdownFilters.jsx

```diff
@@ -5,21 +5,23 @@ import { requiredWhen } from './utils/propTypesUtils';
 
 /** The first filter will be as an input, additional filters will be available in a dropdown.  */
 const DropdownFilters = ({
-  columns,
+  columns, noOfBreakoutFilters,
```

Comment:
Opposed to `numBreakoutFilters` ?

Reply:
Not at all, I like it better. Naming things is hard.